.gitignore

```diff
@@ -27,7 +29,6 @@ notebooks/.ipynb_checkpoints/
 
 flexmeasures/ui/static/documentation
 documentation/img/screenshot_*
-migrations/dumps
```

Comment:
Why?

Reply:
Who knows. This is old. I can remove it.

Reply:
My line of thinking is that this looks like it prevents database dumps from accidentally being pushed to GH. We also have created migrations that create a dump file upon upgrading, to support downgrading without data loss. I imagine they use this folder for storing those.

Reply:
Who is "they"?

Reply:
I've edited my comment for clarification.

Reply:
But.. you didn't?

Reply:
I forgot that I would have to actively do something to leave it.